app/models/mixins/authentication_mixin.rb

```diff
@@ -12,6 +12,33 @@ def self.authentication_check_schedule
       assocs = zone.respond_to?(assoc) ? zone.send(assoc) : []
       assocs.each { |a| a.authentication_check_types_queue(:attempt => 1) }
     end
+
+    def self.queue_authentication_check(args, user_id, zone)
```

Comment:
@jntullo Ok, that is good.

Any idea what is the average wait time for the UI before it gets the validation results?
(assuming the queued validation operation is the first task on the queue)